src/relentless/simulate/simulate.py

```diff
@@ -566,12 +566,18 @@ class PairPotentialTabulator(PotentialTabulator):
         potential.
     neighbor_buffer : float
         Buffer radius used in computing the neighbor list.
+    exclusions : list
+        The neighborlist nominally includes all pairs within ``rmax`` of each other.
+        This option allows for exclusions of pairs that should not be included in the
+        neighbor list. The string should be formatted as a tuple of strings, e.g.,
+        ``['bond', '1-3', 1-4]``.
 
     """
 
-    def __init__(self, potentials, start, stop, num, neighbor_buffer):
+    def __init__(self, potentials, start, stop, num, neighbor_buffer, exclusions=[]):
```

Comment:
This is one of those Python quirks you learn not to do the hard way. =) I would default this to None:
```suggestion
    def __init__(self, potentials, start, stop, num, neighbor_buffer, exclusions=None):
```
then reinterpret it as an empty list internally.

You can read here about why:

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Thanks! I hadn't seen that before! 😅